### PR TITLE
Feature routing

### PR DESCRIPTION
**Pull request by shruthihsrinath**

### CODE REVIEW

File: src/app/collaboration/collaboration.component.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <article class="collab-page">
     <section>
-        <h2>
+        <h2 style="margin: 0;">
             {{collabName}}
         </h2>
-        <hr class="seperator" />
+        <!-- <hr class="seperator" /> -->
```
Comment (shreeshbhat) [inline src/app/collaboration/collaboration.component.html:6]: Remove commented code

File: src/app/portfolio/portfolio.component.html [hunk 2]
```diff
@@ -1,13 +1,14 @@
 <article class="portfolio-page">
-    <h2 class="heading">COLLABORATION</h2>
-    <hr class="seperator" />
+    <h2 class="heading">COLLABS</h2><br />
+    <!-- <hr class="seperator" /><br /> -->
```
Comment (shreeshbhat) [inline src/app/portfolio/portfolio.component.html:3]: Remove commented code

File: src/app/portfolio/portfolio.component.html [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
     <div class="collab-container">
-        <img src="../../assets/sheetal-koskii.jpg" alt="image, for koskii collaboration link" class="collab-image"
-            (click)="onClick($event)">
-        <div class="collab-content">
-            <div class="collab-text">
-                KOSKII <br>
-                <hr class="link-line" />
+        <div class="collab-card" *ngFor="let image of collabData">
+            <img src="{{image.url}}" alt="image, for koskii collaboration link" class="collab-image"
```
Comment (shreeshbhat) [inline src/app/portfolio/portfolio.component.html:6]: tag shouldn't have click listeners on it. It doesn't make the experience accessible.

File: src/app/portfolio/portfolio.component.ts [hunk 4]
```diff
@@ -1,13 +1,26 @@
...
   selector: 'app-portfolio',
   templateUrl: './portfolio.component.html',
   styleUrls: ['./portfolio.component.css']
 })
 export class PortfolioComponent {
-  constructor() { }
-  onClick(event: any) {
-    console.log(event.target.alt);
+
+  collabData: any = [
```
Comment (shreeshbhat) [inline src/app/portfolio/portfolio.component.ts:11]: Define a type instead of any

File: src/styles.css [hunk 5]
```diff
@@ -12,8 +12,27 @@ html {
     font-family: 'Comfortaa', cursive;
     font-size: 20px;
     color: black;
+
+    --scrollbarBG: rgba(232, 82, 82, 0.179);
+    /* --scrollbarBG: rgba(225, 225, 225, 0.19); */
```
Comment (shreeshbhat) [inline src/styles.css:17]: Remove commented code

File: src/app/portfolio/portfolio.component.html (hunk 2 above)
Comment (shruthihsrinath) [reply src/app/portfolio/portfolio.component.html:3]: Removed commented code.

File: src/app/portfolio/portfolio.component.ts (hunk 4 above)
Comment (shruthihsrinath) [reply src/app/portfolio/portfolio.component.ts:11]: changed in feature-data-service branch

**Review by shreeshbhat — APPROVED** [review]

**Pull request merged**